querybook/migrations/versions/767cf0f573bd_add_table_links_to_datatable.py

```diff
@@ -0,0 +1,84 @@
+"""add table links to datatable
```

Comment:
You should probably edit this file to only include the changes made in this PR (aka add column table_links for upgrade and drop_column table_links for downgrade)